docs/docs/widgets/code-editor.md

```diff
@@ -2,26 +2,25 @@
 id: code-editor
 title: Code Editor
 ---
-# Code Editor
 
-**Code Editor** widget is a versatile text editor for editing code and supports several languages. 
+**Code Editor** component is a versatile text editor for editing code and supports several languages. 
 
 <div style={{textAlign: 'center'}}>
 
-<img className="screenshot-full" src="/img/widgets/code-editor/editor.png" alt="ToolJet - Widget Reference - Code editor" />
+<img className="screenshot-full" src="/img/widgets/code-editor/editor.png" alt="ToolJet - Component Reference - Code editor" />
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
 :::info
-Any property having `Fx` button next to its field can be **programmatically configured**.
+Any property having **fx** button next to its field can be **programmatically configured**.
 :::
 
-| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
-|:----------- |:----------- |:----------------- |
+| <div style="width: 100px;">Property</div> | <div style="width: 100px;">Description</div> | <div style="width: 135px;">Expected Value</div> |
```

Comment:
hey there is this syntax error with all the style property 
it should be like `<div style={{ width:"100px"}}> Property </div>`  instead of ` <div style="width: 100px;">Property</div>`

Please update this with all the style properties in both versions.

Reply:
When I preview the markdown file in VSCode the suggested changes result in in the screenshot attached - this has led me to change it the way I did. 

However, I understand the VSCode extension may render markdown differently. I just wanted to explain my changes, and confirm your suggested changes before I change and push the commit. 

![image](https://github.com/user-attachments/assets/eacf7f14-2e45-462a-850e-df6b0fcc11eb)

Reply:
No worries!
But please update the syntax as it was before so that it don't cause any error on the live site.
Also if you have locally setup the ToolJet documentation then you can use `npm run start` command to check the changes on your local.